doc/data/messages/i/import-error/good.py

```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+from pathlib import Path
+
+import sphinx
```

Comment:
```suggestion
# import third_party
```
I don't think we can demonstrate this one easily. Might be better to just keep the typo only. If a beginner encounter ``# import third_party `` it will be confusing for them.

Reply:
Makes sense to me, updated to keep the typo only.